shared/fixes/bash/accounts_authorized_local_users_sidadm_orasid.sh

```diff
@@ -45,14 +48,14 @@ if [ -d "/sapmnt" ] ; then
 	done
 
 	# if owner of any brspace file is oracle, add oracle to the userlist
-	if test "$(is_owner_oracle "${path_to_brspace_list[@]}")" = 1 ; then userlist="$userlist|oracle" ; fi
```

Comment:
`path_oracle_SID_list` is space-separated string, so pass just "$path_to_brspace_list".